blog/2022/12-19-from-cloud2sql/index.md

```diff
@@ -0,0 +1,117 @@
+
+An example for an individual `aws_elb` would look like this:
+
+```json
+{
+  "id": "hgoa3433hOo7j6pEUDmozE",
+  "type": "node",
+  "revision": "_fH7wa0K--I",
+  "reported": {
+    "kind": "aws_elb",
+    "id": "blogtest-apiserver-526928195.us-west-2.elb.amazonaws.com",
+    "name": "blogtest-apiserver",
+    "ctime": "2022-11-22T09:14:30Z",
+    "tags": {
+      "owner": "lukas",
+      "sigs.k8s.io/cluster-api-provider-aws/role": "apiserver",
+      "sigs.k8s.io/cluster-api-provider-aws/cluster/blogtest": "owned",
+      "Name": "blogtest-apiserver"
+    },
+    "lb_type": "elb",
+    "backends": [
+      "i-08381208b144af211"
+    ],
+    "scheme": "internet-facing",
+    "elb_canonical_hosted_zone_name": "blogtest-apiserver-526928195.us-west-2.elb.amazonaws.com",
+    "elb_canonical_hosted_zone_name_id": "Z2H1FL2FABSF3",
+    "elb_listener_descriptions": [
+      {
+        "listener": {
+          "protocol": "TCP",
+          "load_balancer_port": 6443,
+          "instance_protocol": "TCP",
+          "instance_port": 6443
+        },
+        "policy_names": []
+      }
+    ],
+    "elb_policies": {
+      "app_cookie_stickiness_policies": [],
+      "lb_cookie_stickiness_policies": [],
+      "other_policies": []
+    },
+    "elb_backend_server_descriptions": [],
+    "elb_availability_zones": [
+      "us-west-2c",
+      "us-west-2b",
+      "us-west-2a"
+    ],
+    "elb_health_check": {
+      "target": "SSL:6443",
+      "interval": 10,
+      "timeout": 5,
+      "unhealthy_threshold": 3,
+      "healthy_threshold": 5
+    },
+    "elb_source_security_group": {
+      "owner_alias": "999264467951",
+      "group_name": "blogtest-apiserver-lb"
+    }
+  }
+}
+```
+
+While a graph data model is well suited to representing the complex and hierarchical data collected by Resoto, the reality is that many people are familiar with SQL and the traditional database model based on rows and columns. SQL is a mature and well-established language, with a rich ecosystem of tools and applications.
+
+To provide users with the option to use SQL with the data collected by Resoto, we developed [cloud2sql](https://cloud2sql.com), a sub-project of Resoto. cloud2sql is based on the same rich data collected by Resoto, but it flattens that data into tables, complete with foreign keys and link tables that represent the relationships between different cloud resources.
+
+![Flattened AWS ELB](./img/aws_elb_flattened.png)
+
+This allows users who are familiar with SQL to easily work with the data collected by Resoto, using the tools and applications that they are already familiar with. It also makes it easy to integrate the data collected by Resoto into existing SQL-based workflows and processes.
+
+![AWS ELB Summary](./img/aws_elb_summary.png)
+
+[cloud2sql](https://cloud2sql.com) is stateless and runs standalone, independently of Resoto. It is however based on the same collector plugins that Resoto uses to gather data.
```

Comment:
I wonder if we should also add a sentence saying that we are currently integrating this option into Resoto. wdyt?